spec/requests/api/v3/providers/suggest_provider_spec.rb

```diff
@@ -79,23 +79,17 @@
     expect(JSON.parse(response.body)["data"].length).to eq(10)
   end
 
-  it "returns bad request if query is empty" do
+  it "returns status: bad request if query is empty" do
     get "/api/v3/provider-suggestions"
 
     expect(response.status).to eq(400)
   end
 
-  it "returns bad request if query is too short" do
+  it "returns status: bad request (400) if query is too short" do
     provider
 
     get "/api/v3/provider-suggestions?query=#{provider.provider_name[0, 2]}"
 
     expect(response.status).to eq(400)
   end
-
-  it "returns bad request if start of query is not alphanumeric" do
-    get "/api/v3/provider-suggestions?query=%22%22%22%22"
-
-    expect(response.status).to eq(400)
-  end
```

Comment:
can you invert this test instead of deleting it?

"should not return bad request if start of query is not alphanumeric"

Reply:
I don't see the point in testing default behaviour?

Reply:
see below